api/models/models.js

```diff
@@ -144,22 +145,30 @@ exports.addCommentByArticleId = async (id, username, body) => {
   }
 };
 
-exports.retireCommentByCommentId = async (id) =>{
+exports.retireCommentByCommentId = async (id) => {
   try {
-  const values = [id];
-  const queryStr = `DELETE FROM comments WHERE comment_id = $1 RETURNING *`;
-  const data = await db.query(queryStr, values);
-  
-  if (!data.rows.length) {
-    await checkExists("comments", "comment_id", id);
-  }
+    const values = [id];
+    const queryStr = `DELETE FROM comments WHERE comment_id = $1 RETURNING *`;
+    const data = await db.query(queryStr, values);
 
+    if (!data.rows.length) {
+      await checkExists("comments", "comment_id", id);
+    }
   } catch (err) {
     if (err instanceof CustomError) throw err;
     throw new CustomError(500, null, null, err);
   }
+};
+exports.fetchEndpoints = async () => {
+  try {
+    const path = `${__dirname}/../../endpoints.json`;
+    const endpoints = await fs.readFile(path, "utf-8");
+    return JSON.parse(endpoints);
```

Comment:
If you require a JSON file it is automatically read as a JavaScript object and so you dont need to use readFile/parse it :)